package.json

```diff
@@ -29,7 +29,7 @@
     "firebase-admin": "^8 || ^9"
   },
   "devDependencies": {
-    "@firebase/testing": "^0.20.11",
+    "@firebase/rules-unit-testing": "^1.0.5",
```

Comment:
I didn't even notice when they switched this in the docs - was there a warning or error message that lead you to update this or was it just reading it in the docs?

Reply:
Without it, the GeoPoint class is not generated from firesbase-admin and is considered to be a different class, so we investigated and found that `@firebase/testing` is deprecated.